src/components/all/all.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import styled from "styled-components";
+
+import Shoe from "../styled/shoe/shoe";
+import icon__scissor from "../../../public/assets/icon__scissor.svg";
+import icon__plus from "../../../public/assets/icon__plus.svg";
+
+const CraftList = styled.ul.attrs({
+	className: "list pl0 flex flex-wrap mh4"
+}) `
+margin-top: 5rem;
+`;
+
+const CraftItem = styled.li.attrs({
+	className: "w-30 h4 mh1"
+}) ``;
+
+const Plus = styled.button.attrs({
+	className: "fixed bottom-0 z-2"
+}) `
+	left: 50%;
+	transform: translateX(-50%);
+`;
+
+const getShoeDetails = (shoeObj) => {
```

Comment:
This could be put inside the `utilities` folder and perhaps unit tested?

Reply:
Actually instead of this function, perhaps use deconstruction, something like:
`<Shoe style={ {craft.details} } width="100%" />`